core/workspace_svg.js

```diff
@@ -537,53 +544,42 @@ Blockly.WorkspaceSvg.prototype.render = function() {
 };
 
 /**
- * Turn the visual trace functionality on or off.
- * @param {boolean} armed True if the trace should be on.
+ * Was used back when block highlighting (for execution) and block selection
```

Comment:
Existing users already need to call highlight directly.  The call to traceOn was designed to deal with the UI collision of using selection as an execution marker.  Now that they are separate, this call does nothing and can be deleted.  There is no API change for existing users.